src/TableUtils.jl

```diff
@@ -98,21 +136,19 @@ function create_table(
 
     # Cutoff headers if needed:
     cutoff = 14
-    headers = String[
-        if length(head) <= cutoff
-            head
-        else
-            first(head, cutoff) * "..."
-        end for head in headers
-    ]
+    headers = String[if length(head) <= cutoff
+        head
+    else
+        first(head, cutoff) * "..."
+    end for head in headers]
```

Comment:
Why is this bit changed?

Reply:
The logic hasn't changed but some things were formatted by `JuliaFormatter.jl`